fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/provider/foundationdb/IndexingThrottle.java

```diff
@@ -144,6 +145,22 @@ boolean mayRetryAfterHandlingException(@Nullable FDBException fdbException,
             return true;
         }
 
+        private static boolean shouldLessenWork(@Nullable FDBException ex) {
+            // These error codes represent a list of errors that can occur if there is too much work to be done
+            // in a single transaction.
+            if (ex == null) {
+                return false;
+            }
+            final Set<Integer> lessenWorkCodes = new HashSet<>(Arrays.asList(
```

Comment:
This set can be declared as a static instance - no need to instantiate every time